packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -226,9 +232,27 @@ class _FlutterValidator extends DoctorValidator {
   }
 }
 
+class NoIdeValidator extends DoctorValidator {
+  NoIdeValidator() : super('Flutter IDE Support');
+
+  @override
+  Future<ValidationResult> validate() async {
+    // TODO(danrubel) do not show Atom once IntelliJ support is complete
```

Comment:
Once the IntelliJ plugin is available (and at ~parity with the Atom plugin), we'll retire the flutter atom plugin, and the dart atom plugin will move to being community supported.